slo_generator/exporters/prometheus_remote_write.py

```diff
@@ -0,0 +1,244 @@
+"""
+`prometheus_remote_write.py`
+Prometheus Remote Write exporter class.
+"""
+import logging
+import re
+import time
+from typing import Dict, Tuple
+
+import requests
+import snappy
+from slo_generator.exporters.base import MetricsExporter
+from slo_generator.exporters.gen.remote_pb2 import (
```

Comment:
ditto, this import will become `from slo_generator.proto.remote_pb2 import` after moving the generated files to the proto folder